packages/material/test/renderers/MaterialArrayControl.test.tsx

```diff
@@ -309,4 +351,118 @@ describe('Material array control', () => {
     const nrOfRows = wrapper.find('tr').length;
     expect(nrOfRows).toBe(0);
   });
+  it('should render sort buttons if showSortButtons is true', () => {
+    const data = { test: ['foo'] };
+    const store = initJsonFormsStore();
+    store.dispatch(Actions.init(data, fixture2.schema, fixture2.uischema));
+    wrapper = mount(
+      <Provider store={store}>
+        <MaterialArrayControlRenderer
+          schema={fixture2.schema}
+          uischema={fixture2.uischema}
+        />
+      </Provider>
+    );
+    const btnCount = wrapper.find('button').length;
+    // add button
+    // delete button
+    // two dialog buttons (no + yes)
+    // sort Buttons (up + down)
+    expect(btnCount).toBe(6);
+  });
+  it('should be able to move item down if down button is clicked', () => {
+    const store = initJsonFormsStore();
+    store.dispatch(
+      Actions.init(fixture2.data, fixture2.schema, fixture2.uischema)
+    );
+    wrapper = mount(
+      <Provider store={store}>
+        <MaterialArrayControlRenderer
+          schema={fixture2.schema}
+          uischema={fixture2.uischema}
+        />
+      </Provider>
+    );
+    //first row is header in table
+    // first buttton is up arrow, second button is down arrow
+    const downButton = wrapper
```

Comment:
can't we find the button using the aria label? this would make the test more robust against layout changes